modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -0,0 +1,417 @@
+<?php
+/**
+ * Candidate parameters form handler
+ *
+ * Handles form update actions received from a front-end ajax call
+ *
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Media
+ * @author   Stella L. <slee.mcin@gmail.com>
+ * @license  Loris license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+if (isset($_POST['tab'])) {
+    $tab = $_POST['tab'];
+
+    $db   =& Database::singleton();
+    $user =& User::singleton();
+
+    if ($tab == "candidateInfo") {
+        editCandInfoFields($db, $user);
+    } else if ($tab == "probandInfo") {
+        editProbandInfoFields($db, $user);
+    } else if ($tab == "familyInfo") {
+        editFamilyInfoFields($db, $user);
+    } else if ($tab == "deleteFamilyMember") {
+        deleteFamilyMember($db, $user);
+    } else if ($tab == "participantStatus") {
+        editParticipantStatusFields($db, $user);
+    } else if ($tab == "consentStatus") {
+        editConsentStatusFields($db, $user);
```

Comment:
I think this would be cleaner if each tab had it's own script instead of mixing everything together into one (but I don't think that's worth blocking the merge for.)